RecoVertex/PrimaryVertexProducer/src/DAClusterizerInZ_vect.cc

```diff
@@ -1027,8 +1053,9 @@ vector<TransientVertex> DAClusterizerInZ_vect::vertices(const vector<reco::Trans
       y.z_ptr[k] = 0;
     }
 
+  const auto z_sum_init = rho0 * local_exp(-beta * dzCutOff_ * dzCutOff_);
```

Comment:
cf IntZT_vect
```C++
const auto zsuminit = rho0 * local_exp(-beta * dzCutOff_ * dzCutOff_);
```
it would be nice to keep consistent naming.

Reply:
OK